comp/logistics.yaml

```diff
@@ -0,0 +1,51 @@
+summary: Organise the logistics
+
+priority: should
+
+component: Competition
+
+milestone: $SRYYYY Competition Prep
+
+area-owner: event-logistics
+
+description: >-
+  To run the competition each year, we need a set of equipment.
+  This is a combination of items that can be reused each year, stored in the container, and consumables that we may need to purchase.
+
+
+  Well before the competition we'll need to visit storage, to check quantities of consumables (tape etc.) and check the condition of other items.
+  We should also create a detailed list of the quantities and competition locations that they need to be delivered to.
+
+
+  The items we should check for are:
```

Comment:
This list is quite vast. Should we split it into "things we need to make sure we collect" vs "consumables we need to buy"?

Reply:
No, the outcome of checking both consumables and non-consumables is either purchasing replacements or it's all good.